homework-1/cash_register.py

```diff
@@ -0,0 +1,86 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+Your class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+
+
+class CashRegister:
+
+    def __init__(self):
+        self.total_items = dict()
+        self.total_price = 0
+        self.discount = 0
+
+    def add_item(self, item, price):
+        self.total_items[item] = price
+        return self.total_items
+
+    def remove_item(self, item):
+        self.total_items.pop(item)
+        return self.total_items
+
+    def apply_discount(self, item, discount_applicable, discount):
+        if discount_applicable:
+            discounted_price = self.total_items[item]*(1-discount/100)
+            print(f"{item}: £{'%.2f' % round(discounted_price,2)}")
+            self.total_items[item] = discounted_price
+        else:
+            print(f"{item}: £{'%.2f' % round(self.total_items[item],2)}")
+
+
+    def get_total(self):
+        print(f"The total cost is £{'%.2f' % round(sum(self.total_items.values()),2)}")
```

Comment:
Since the method is called `get_total` rather than `view_total` or similar, it might make more sense to return the total in case it was ever needed for further processing